### PR TITLE
Install Pytest tool and rename flake8.yml

### LINKED ISSUE #57: Installing Pytest
Installing Pytest files as part of Github actions tool

### PR DESCRIPTION
**Pull request by noaramon**
Adding Pytest as part of Github actions in order to assist with tests on our data models.
This PR also contains renaming to flake8.yml file.

fixes #57

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ steps:
+ - name: Install Pipenv and Git
+ run: dnf install -y pipenv git
+ - name: Checkout code
+ uses: actions/checkout@v2
+ - name: Setup environment
+ run: pipenv sync --dev
+ - name: Run PyTest
+ run: pipenv run pytest -v
+
```
Comment (ravidbro) [inline .github/workflows/pytest.yml]: Now you have an extra line, you shouldn't see here the red symbol

**Comment by ravidbro** [comment]
my concern is that I see only the flake8 check running on this PR

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (Noa-Git) [reply .github/workflows/pytest.yml]: I saw that and immediately modified the file, but I can see the change in the File Changed section

**Comment by noaramon** [comment]
Solved by hand-writing the code.

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (noaramon) [reply .github/workflows/pytest.yml]: I saw that and immediately modified the file.

**Comment by ravidbro** [comment]
You can see that now, it's running.
The reason is the number of spaces before you begin the row, yaml format requires two spaces

**Comment by ravidbro** [comment]
Now that it's running you can see that the test fail

**Comment by simzacks** [comment]
@ravidbro the tests aren't failing, there are no tests to execute. There are a few tests.py files in the repo, but they are all blank.
The failure is Exit Code 5 which means "No tests were collected"

**Review by ravidbro — APPROVED** [review]

**Pull request merged**